DataVaildation/DataVaildation.robot

```diff
@@ -0,0 +1,131 @@
+*** Settings ***
+Library  DatabaseLibrary
+Library    Collections
+
+
+*** Variables ***
```

Comment:
can you avoid adding new variables compared to the sample implementation (especially for DB setup) since most of these are injected in, just use the pre-existing implementation